server/src/test/java/org/elasticsearch/snapshots/DedicatedClusterSnapshotRestoreIT.java

```diff
@@ -981,6 +985,34 @@ public void testRestoreShrinkIndex() throws Exception {
         ensureYellow();
     }
 
+    public void testSnapshotWithDateMath() {
+        final String repo = "repo";
+        final AdminClient admin = client().admin();
+
+        final IndexNameExpressionResolver nameExpressionResolver = new IndexNameExpressionResolver(Settings.EMPTY);
+        final String snapshotName = "<snapshot-{now/d}>";
+        final String expression = nameExpressionResolver.resolveDateMathExpression(snapshotName);
+
+        logger.info("-->  creating repository");
+        assertAcked(admin.cluster().preparePutRepository(repo).setType("fs")
+            .setSettings(Settings.builder().put("location", randomRepoPath())
+                .put("compress", randomBoolean())));
+
+        logger.info("-->  creating date math snapshot");
+        CreateSnapshotResponse snapshotResponse =
+            admin.cluster().prepareCreateSnapshot(repo, snapshotName)
+                .setIncludeGlobalState(true)
+                .setWaitForCompletion(true)
+                .execute().actionGet();
```

Comment:
just `.get()` will do. Also no need for `setIncludeGlobalState(true)`, that's irrelevant for the test.

Reply:
agreed, just copy-n-pasted from another place